tensorboard/plugins/debugger/tf_debugger_dashboard/tf-debugger-dashboard.html

```diff
@@ -324,6 +324,7 @@ <h2>Continue...</h2>
   </template>
   <script src="tensor-shape-helper.js"></script>
   <script>
+    const _DEFAULT_LEFT_MENU_WIDTH = 450;
```

Comment:
Is it possible to make this a percentage instead of absolute width in pixels, to better accommodate browser windows of different sizes?

Reply:
I'm not sure if a percentile min is desirable. Let me know your thoughts.

We don't want the table on the left pane to be less than 450px - I think the table looks subpar when it is. Furthermore, if a user has a very wide screen, we don't want to enforce a large minimum width for the left pane - instead, the user should be able to commit most screen real estate to the main content.

Reply:
Sounds good!